samples/msal-node-samples/ElectronSystemBrowserTestApp/src/AuthProvider.ts

```
@@ -60,7 +60,7 @@ export default class AuthProvider {
                 .removeAccount(this.account);
             this.account = null;
         } catch (error) {
-            console.log(error);
+            console.error(error);
```

I don't think catching excpetions and silently logging them is a good pattern. It's better to let the errors bubble up.

---

This is out of scope for this PR. We can address this in a follow-up PR.

---

As discussed with @tnorling , let's fix the codeQL issues by removing these pointless console.error calls instead of sanitizing the error messages.